vuepress/docs/.vuepress/components/SpecialLayout.vue

```diff
@@ -76,7 +76,18 @@
       </div>
     </div>
     <div class="main-content">
-      <div class="begin-developing layout-container">
+      <div v-if="jhipster" class="layout-container" >
+        <div class="get-started-grid width-container">
+          <div class="get-started-left jhipster">
+            <img src="./assets/logo-jhipster.svg" alt="jhipster"/>
+            JHipster
+          </div>
+          <div class="get-started-right">
+            <p>Greetings, Java Hipster! See the links below to get started with your Entando journey or click <a href="/v6.1/tutorials/backend-developers/generate-microservices-and-micro-frontends.html">here</a> to learn how Entando leverages JHipster to generate micro frontends and microservices.</p>
```

Comment:
We'll have to upgrade this to 6.2 at release

Reply:
Yep, there are several paths like that we'll need to update.